plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotIntegrationSmokeTest.java

```diff
@@ -95,7 +95,7 @@ protected QueryRunner createQueryRunner()
                     createTestRecord(
                             Arrays.asList("string_" + (offset), "string1_" + (offset + 1), "string2_" + (offset + 2)),
                             Arrays.asList(false, true, true),
-                            Arrays.asList(54, -10001, 1000),
+                            Arrays.asList(54 + i / 3, -10001, 1000),
```

Comment:
Is this to get a fractional value?

Reply:
Also I'm confused by the fact that this column has integers, booleans, string and fractional values.

What should I look at to figure out how they actually land into Pinot? The table config?

Reply:
This is just to have more distinct values in the column instead of just `54`, it is an integer so the values would be `54`, `54`, `54`, `55`, `55`, `55`, etc.